File: IronMan/backend/src/models/OrderModel.js
const mongoose = require('mongoose');
const OrderSchema = mongoose.Schema({
    UserID:{
        type:String,
        //required:true,
    },
    UserName:{
        type:String,
        //required:true,
    },
    ShopName:{
        type:String,
        //required:true,
    },
    OrderDateTime:{
        type:String,
        //required:true,
    },
    Services:{
        type:Object,
        //required:true,
    },
    Cost:{
        type:String,
        //required:true,
    },
});
const OrderModel = mongoose.model("Order",OrderSchema);
module.exports = OrderModel;

File: IronMan/ironman_delivery/lib/models/ordermodel.dart
class Ordermodel {
  Ordermodel(
      {required this.cost,
      required this.orderNumber,
      required this.orderStatus,
      required this.service,
      required this.id});
  String orderNumber, orderStatus, id;
  Map<String, Map<String, dynamic>> service;
  String cost;
  factory Ordermodel.fromJson(Map<String, dynamic> json) {
    return Ordermodel(
      id: json['UserID'] as String,
      cost: json['Cost'] as String,
      orderNumber: json['orderNumber'] as String,
      orderStatus: json['OrderStatus'] as String,
      service: Map<String, Map<String, dynamic>>.from(json['Services']),
    );
  }
}
